pom.xml

```diff
@@ -14,6 +14,7 @@
 
   <properties>
     <marketplace.duoweb.authentication.version>1.7.0-SNAPSHOT</marketplace.duoweb.authentication.version>
+    <nuxeo.target.version>11.1</nuxeo.target.version>
```

Comment:
Not sure we want to hardcode `11.1` here, but keep `{nuxeo.distribution.version}` by default, and change it when releasing the MP if needed?
WDYT @ffjdm?

Reply:
This is needed to handle package matching the original release instead of the HF one (that is changed automatically when releasing the MP)

Reply:
After discussing with @troger the best way is to have the decoupling but use the same version (i.e. 11.1-SNAPSHOT for both variables). The values will differ only when releasing if needed.

Reply:
Yes, keep `11.1-SNAPSHOT` here, having `11.1` means that the SNAPSHOT package is available and can be installed on a 11.1 release.